PROVISIONING.md

```diff
@@ -0,0 +1,72 @@
+# Discovery and Provisioning
+
+## Docs
+- please look at https://github.com/silicom-ltd/uBMC
+- please look at https://opencomputeproject.github.io/onie/overview/
+- please look at https://fidoalliance.org/intro-to-fido-device-onboard/
+- please look at https://www.rfc-editor.org/rfc/pdfrfc/rfc8572.txt.pdf (Secure Zero Touch Provisioning (SZTP))
+- please look at https://pypi.org/project/sztpd/
+- please look at https://watsen.net/docs/sztpd/0.0.11/admin-guide/
+
```

Comment:
add also link to https://opencomputeproject.github.io/onie/overview/#design-overview